worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -281,7 +285,10 @@ protected void switchServingState() {
         trimAllListeners();
         break;
       case NONE_PAUSED:
-        logger.info("Trigger action: RESUME PUSH and REPLICATE");
+        long pauseSpendMills = System.currentTimeMillis() - pauseStartTime;
+        logger.info(
+            "Trigger action: RESUME PUSH and REPLICATE, pause push spent: " + pauseSpendMills);
+        pausePushDataTime.add(pauseSpendMills);
```

Comment:
Consider the corner case that a worker enters the pause state but never goes out, then  `pausePushDataTime` will never increase. Maybe we can increase `pausePushDataTime` every N successive pause states, as well as state changes from pause to non_pause.

Also, I think we should consider worker under high load only when the pause time exceeds some threshold, instead of whenever it enters pause state, as https://github.com/apache/incubator-celeborn/pull/1840 does, cc @pan3793

Reply:
> Consider the corner case that a worker enters the pause state but never goes out, then pausePushDataTime will never increase. Maybe we can increase pausePushDataTime every N successive pause states, as well as state changes from pause to non_pause.

This is a good idea.